README.md

```diff
@@ -64,8 +64,108 @@ MySQL password should be the same for initialSetupTests as openmrs password
 ### Running Platform Upgrade test
 - `npm run coreUpgrade`
 
-### Running microfrontend tests
-- `npm run microfrontend`
+---
+# RefApp 3.x  E2E tests
+
+## Setting up the project
+
+1. Clone the project
+    ```
+     git clone git@github.com:openmrs/openmrs-contrib-qaframework.git
+     cd openmrs-contrib-qaframework
+    ```
+1. Install the dependencies
+  ```
+  npm install
+  ```
+You don’t need to set up an OpenMRS instance since we use a [cloud instance](https://openmrs-spa.org/openmrs/spa) for testing.
+
+## Running tests
+
+There are two ways of running tests:
+- **Running with cypress runner**
+  1. Open the Cypress runner
+    ```
+    cypress open
+    ```
+  1. Pick a test from the GUI
+- **Running in command line**
+  
+    Run the desired test using npm run. See package.json
+      
+    ex: `npm run refApp3Login`
+
+## File structure
+```
+.
+├── cypress
+│   ├── fixtures // Test fixtures (e.g. attachments)
+│   │   └── test_image.jpeg
+│   ├── integration
+│   │   └── cucumber
+│   │       └── step_definitions
+│   │           ├── refapp-2.x
+│   │           │   └── login.js
+│   │           └── refapp-3.x // Cypress tests for the refapp 3.x
+│   │               ├── 01-login
+│   │               │   └── login.js
+│   │               ...
+│   ├── plugins
+│   │   └── index.js
+│   ├── support
+│   │   ├── commands.js // Custom commands for Cypress
+│   │   └── index.js
+│   ├── videos  // Screen recordings (set "video": true in cypress.json)
+│   └── tsconfig.json
+├── src
+│   └── test
+│       ├── java
+│       └── resources
+│           ├── features
+│           │   ├── platform
+│           │   ├── refapp-2.x
+│           │   └── refapp-3.x // Cucumber feature files for the refapp 3.x
+│           │       ├── 01-login
+│           │       │   └── login.feature
+│           │       ...
+├── target
+├── README.md
+├── cypress.json // Cypress configuration file
+├── package.json
+├── pom.xml
+```
+
+
+## Writing a new test
+1. Create a new directory with your feature file under `/src/test/resources/features/refapp-3.x/`.
+    
+    The name of the directory should be `<sequence>-<name>`.
+    
+    [Simple style guide](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/src/test/resources/features/refapp-2.x/stylesGuide.feature)
```

Comment:
Is this supposed to be an example feature file? "Simple style guide" doesn't mean anything to me in this context.

Reply:
Yeah, that makes sense. Let me fix it.